src/Sylius/Behat/Context/Setup/CatalogPromotionContext.php

```diff
@@ -404,26 +475,59 @@ public function thereIsAnotherCatalogPromotionAvailableInChannelsThatReducesPric
     }
 
     /**
-     * @Given /^there is another catalog promotion "([^"]*)" available in ("[^"]+" channel) that reduces price by ("[^"]+") and applies on ("[^"]+" variant)$/
+     * @Given /^there is (?:a|another) catalog promotion "([^"]*)" between "([^"]+)" and "([^"]+)" available in ("[^"]+" channel) that reduces price by ("[^"]+") and applies on ("[^"]+" taxon)$/
      */
-    public function thereIsAnotherCatalogPromotionAvailableInChannelThatReducesPriceByAndAppliesOnVariant(
+    public function thereIsACatalogPromotionBetweenAvailableInChannelThatReducesPriceByAndAppliesOnTaxon(
```

Comment:
This may look strange, I removed entirely `there is another catalog promotion`  method and merged it above with `there is (?:a|another)` thus both methods were the same. Git shows that I changed the variant for a taxon, but in fact, this is a new method